app/components/anchor/dialog_component.rb

```diff
@@ -1,8 +1,11 @@
 module Anchor
   class DialogComponent < Component
-    renders_one :show_button, -> do
-      ButtonComponent.new(data: { action: "dialog#showModal" })
-    end
+    renders_one :show_button, lambda { |**kwargs|
+      ButtonComponent.new(
+        data: { action: 'dialog#showModal' },
+        **kwargs
+      )
+    }
```

Comment:
Not sure if it’s desirable or if we have a stylistic preference as an org, but I think we can keep the arrow syntax:

```suggestion
    renders_one :show_button, ->(**kwargs) do
      ButtonComponent.new(
        data: { action: 'dialog#showModal' },
        **kwargs
      )
    end
```

I’m fine with either way!